docs/reference/contributing/guidelines/workflow.md

```diff
@@ -1,42 +1,60 @@
 
-All code changes and additions to Mbed OS are handled through GitHub. If you want to contribute, either by adding features or by fixing bugs, please follow the guidelines for [new features](#contributing-new-features-to-mbed-os) and [bugs](#reporting-and-fixing-bugs). In both cases, please follow the [code style guide and GitHub pull request guidelines](/docs/v5.7/reference/guidelines.html#style)</a>. Please also read the [CLA](/docs/v5.7/reference/guidelines.html#cla) guidelines because we will immediately close pull requests submitted without a CLA.
+#### Gatekeepers
 
-#### Contributing new features to Mbed OS
+##### Mbed OS maintainers
 
-Before contributing an enhancement (new feature, new port and so on), please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on a related feature.
+The maintainers are a small group of Mbed OS engineers who are responsible for the Mbed OS codebase. Their primary role is to progress contributions, both internal and external, from the initial pull request state through to released code. 
+
+Responsibilities:
 
-Patch contributions can only be accepted through GitHub by creating a pull request from forked versions of our repositories. This allows us to review the contributions in a user friendly and reliable way, under public scrutiny.
+1. Check for CLA compliance.
+1. Ensure the relevant stakeholders review pull requests.
+1. Guide contributors both technically and procedurally.
+1. Run pull requests through the CI systems.
+1. Put label version.
+1. Merge pull requests into the requested branches.
+1. Make periodic patch and feature releases.
 
-Please create separate pull requests for each concern; each pull request should have a clear unity of purpose. In particular, separate code formatting and style changes from functional changes. This makes each patch’s true contribution clearer and therefore quicker and easier to review.
+The current maintainers are:
 
-#### Reporting and fixing bugs
+* [Anna Bridge](https://os.mbed.com/users/AnnaBridge).
+* [Martin Kojtal](https://os.mbed.com/users/Kojto).
+* [Jimmy Brisson](https://os.mbed.com/users/theotherjimmy).
+* [Shrikant Tudavekar](https://os.mbed.com/users/shrikant1213).
+* [Sam Grove](https://os.mbed.com/users/sam_grove).
+* [Cruz Monrreal](https://os.mbed.com/users/MrCruz).
 
-Before submitting a bug report or a bug fix, please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on it already.
+##### Mbed OS tech leads
```

Comment:
Can we remove this section. Seems to blur internal organization with an open project and not provide much usefulness.

Reply:
The goal was to define what a team responsible for a module/feature should be doing.

Reply:
I removed it for now.

From experience, we are not always aware what is the release version planned for PR (this can come specified in PR ) plus helping with reviews  (they know who would be the best to do a review)